vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
These certs are issued by a trusted CA (geotrust). Why do they need to be embedded or configured at all? Are you trying to do certificate pinning here?